setup.cfg

```diff
@@ -7,7 +7,7 @@ selenium_driver = chrome
 self-contained-html = true
 
 [bdist_wheel]
-python-tag = py38
+python_tag = py38
```

Comment:
is this even needed? I don't think we support 3.8 (or building wheels for that matter).

Reply:
I don't think so either. Removing.